common4j/src/main/com/microsoft/identity/common/java/net/HttpRequest.java

```diff
@@ -42,10 +43,17 @@
     @Accessors(prefix = "m")
     private final URL mRequestUrl;
 
-    @Getter
     @Accessors(prefix = "m")
     private final byte[] mRequestContent;
 
+    public byte[] getRequestContent() {
```

Comment:
Spotbugs doesn't like the idea of returning the original copy of the array (the value could be modified if an external actor holds a reference to it).

Reply:
Mostly, SpotBugs has good ideas.  It might be good to weigh this one - if you adopt this approach, we're going to want to make very sure that it isn't called repeatedly.  I was going to say - if this was a key or some kind of persistent object state, this is a very good idea to do.  For this one... Given what we do, it might be justified to copy it.